Project1/Task7-2-homework/MobilePhones.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Project1.Task7_2_homework
+{
+     class MobilePhone : Electro, Interface
+    {
+        private double numberOfPixelsInCamera;
+      
+
+        public MobilePhone(double numberOfPixelsInCamera, string? modelName, decimal price)
+        {
+            this.numberOfPixelsInCamera = numberOfPixelsInCamera;
+            this.modelName = modelName;
+            this.price = price;
+        }
+
+        public override string Description
+        {
+            get
+            {
+                return $"Price: {price}, model:{modelName}, number of pixels in camera: {numberOfPixelsInCamera}";
+            }
+        }
+
+        public void TakePhoto()
+        {
+            Console.WriteLine("Press button on the screen and photo is ready");
```

Comment:
the same as above, your can replace it with lambda. If the method contains the only line, so it's better use this kind og coding TakePhoto() => `exression`

Reply:
fix everywhere